src/base/searchengine.cpp

```diff
@@ -672,27 +675,26 @@ QHash<QString, QString> SearchEngine::initializeCategoryNames()
     return result;
 }
 
-qreal SearchEngine::getPluginVersion(QString filePath)
+PluginVersion SearchEngine::getPluginVersion(QString filePath)
 {
     QFile plugin(filePath);
     if (!plugin.exists()) {
         qDebug("%s plugin does not exist, returning 0.0", qPrintable(filePath));
-        return 0.0;
+        return {};
```

Comment:
Honest question: Is there a technical reason to do `return {}` instead of `return PluginVersion`?
To me the 2nd one is more readable.